py/kubeflow/testing/prow_artifacts.py

```diff
@@ -9,7 +9,8 @@
 import os
 import six
 import time
-from google.cloud import storage  # pylint: disable=no-name-in-module
```

Comment:
Since by default, gcp test-infra does not set up `CLOUD_PROVIDER` env var, so this will have no impact on gcp test-infra.

Reply:
/cc @Bobgy

Reply:
FYI, I didn't delete those package imports, I added one if statement to import by situation